.github/workflows/auto_updates.yml

```diff
@@ -64,21 +64,22 @@ jobs:
         # NOTE: The git user name and email used for commits is already configured,
         #       by the crazy-max/ghaction-import-gpg action.
         run: |
-          git commit -a -m "build: Bump poetry.lock dependencies and pre-commit hooks"
+          git commit -a -m "build: Bump `poetry.lock` dependencies and `pre-commit` hooks"
           git push --force origin HEAD:workflow-auto-updates
 
       - name: Create Pull Request
         if: ${{ steps.commit.outcome == 'success' }}
-        uses: actions/github-script@v6
+        uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
         with:
           # This PAT is for the `phylum-bot` account and only has the `public_repo` scope to limit privileges.
           github-token: ${{ secrets.GH_RELEASE_PAT }}
           script: |
-            github.rest.pulls.create({
+            const response = await github.rest.pulls.create({
               owner: context.repo.owner,
               repo: context.repo.repo,
               head: "workflow-auto-updates",
               base: context.ref,
-              title: "build: bump poetry.lock dependencies and pre-commit hooks",
+              title: "build: bump `poetry.lock` dependencies and `pre-commit` hooks",
               body: "Bump dependencies in `poetry.lock` and hooks in `.pre-commit-config.yaml`.",
             });
+            console.log(response);
```

Comment:
I'd assume this log is intentional and not just there for debugging. Or there for debugging, but useful in the future rather than temporary at least.